scico/test/test_fixture.py

```diff
@@ -0,0 +1,15 @@
+import jax
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def run_before_and_after_tests(tmpdir):
+    """Fixture to execute asserts before and after a test is run"""
+    # Setup: fill with any logic you want
+    print("Before: ", jax.config.read("jax_enable_x64"))
+
+    yield  # this is where the testing happens
+
+    # Teardown : fill with any logic you want
+    print("After: ", jax.config.read("jax_enable_x64"))
```

Comment:
Should this be here? Looks like debugging code (though might be useful to have).